components/authz-service/storage/v2/memstore/memstore.go

```diff
@@ -12,20 +13,27 @@ import (
 )
 
 type State struct {
-	policies *cache.Cache
-	roles    *cache.Cache
-	projects *cache.Cache
-	ms       storage.MigrationStatus
+	policies       *cache.Cache
+	roles          *cache.Cache
+	projects       *cache.Cache
+	ms             storage.MigrationStatus
+	policyChangeID int64
```

Comment:
I think if we are going to use AddInt64 we need to ensure this is 64byte aligned.  According to the sync/atomic docs, one way to do that is to make sure this is the first entry in the struct.  Perhaps we could move this to the top with a comment not to move it to protect against hard-to-find bugs in the future.

Reply:
Note this is a supa-nit as I think this code is correct on x86 and likely everything we support currently